package.json

```diff
@@ -50,15 +50,20 @@
     "split2": "^2.1.1",
     "std-mocks": "^1.0.0",
     "through2": "^2.0.0",
-    "winston-compat": "^0.1.1"
+    "winston-compat": "^0.1.1",
+    "babel-cli": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "rimraf": "^2.6.2"
```

Comment:
Please move these three to `devDependencies`.

Reply:
I went to move them but they're already `devDependencies` :)